src/pyasm/search/sql_dumper.py

```diff
@@ -345,6 +357,23 @@ def dump_tactic_inserts(my, path, mode='sql'):
 
             data = sobject.get_data()
             for name, value in data.items():
+                
+                if my.replace_dict:
+                    
+                    for column,replace_args in my.replace_dict.items():
+                        if name == column:
+                            project_code = replace_args[0]
+                            replace_str = replace_args[1]
+                            regex = replace_args[2]
+                            
+                            if not re.match(regex,value):
+                                raise TacticException("%s does not conform to standard format. Expected format must match %s"%(column,regex))
+                            value = re.sub(regex,replace_str,value)
```

Comment:
add an else condition where you just do a simple replacement:
 value = replace_str